EpsilonAuxBmsV2/Tests/tests/SendAuxTripTaskTest/SendAuxTripTaskTest.c

```diff
@@ -0,0 +1,55 @@
+#include "SendAuxTripTaskTest.h"
+#include "unity.h"
+#include "SendAuxTripTask.h"
+#include "Mockcmsis_os2.h"
+
+/*
+checks to made sure that the data from auxTrip is successfully sent
+into the data array of canQueueData
+*/
+
+osMutexId_t auxTripMutex;
+const CAN_TxHeaderTypeDef baseCanTxHdr;
+osMessageQueueId_t canTxGatekeeperQueue;
+
+AuxTrip auxTrip =
+{
+    .chargeTripDueToHighCellVoltage = 1, .chargeTripDueToHighTemperatureAndCurrent = 0,
+    .chargeTripDueToPackCurrent = 0, .dischargeTripDueToLowCellVoltage = 1,
+    .dischargeTripDueToHighTemperatureAndCurrent = 0, .dischargeTripDueToPackCurrent = 1,
+    .protectionTrip = 0
```

Comment:
@badeniran was this suppose to be like 
```
blah blah = 
{ 
    .blah = 0
      .blah = 0
        .blah = 0
};
```
after running `format.sh`?

Reply:
I think if he put each assignment on a new line it would've done that